castle/cms/upgrades/upgrade_2_3_5.py

```diff
@@ -0,0 +1,19 @@
+from plone.registry.interfaces import IRegistry
+from castle.cms.interfaces import ICrawlerConfiguration
+from zope.component import getUtility
+
+import logging
+log = logging.getLogger(__name__)
+
+PROFILE_ID = 'profile-castle.cms.upgrades:2_3_5'
+
+
+def upgrade(context, logger=None):
+    registry = getUtility(IRegistry)
```

Comment:
Instead, you just want to run the upgrade profile.

Reply:
For reference: https://github.com/castlecms/castle.cms/blob/master/castle/cms/upgrades/configure.zcml#L134